### DISCUSSION TITLE
Need help for react-native-firebase/firestore in v7.8.6

### DISCUSSION THREAD

**Discussion by amitmundrag08** [invertase/react-native-firebase#4432] [Q&A]
I am having trouble in fetching data from firestore for android speicifically. Please help me through this. My app is stuck and dependent on this.

**Comment by andersonaddo** [comment]
Provide more information to get proper help from anyone. Sample code, error description, error logs, etc.

**Reply by amitmundrag08** [reply]
Here "lastSyncedServerTimestamp" is the last time when I fetched data from server

firestore().collection("CollectionName").where(
"_serverDocumentUpdatedOn",
">",
new Date(lastSyncedServerTimestamp.valueOf() - 5 * 60000)
) // 5 minutes before the last refresh serverDateTimeStamp to provide for server side time overlap and misses
.orderBy("_serverDocumentUpdatedOn", "desc")
.onSnapshot((change) => {

})

For android I am getting change.type as first removed and then as added when I am modifying data where as for iOS it is working fine.
